Taskfile.yml

```diff
@@ -424,6 +423,23 @@ tasks:
     status:
       - "docker manifest inspect {{.LOCAL_REGISTRY_CONTROLLER_DOCKER_IMAGE}} > /dev/null"
 
+  controller:docker-push-multiarch:
+    desc: Builds, tags and pushes the multi architecture controller container image to repository
+    deps:
+      - controller:bundle-crds
+    dir: "{{.CONTROLLER_ROOT}}"
+    run: always
+    cmds:
+      - 'if [ -z "{{.DOCKER_PUSH_TARGET}}" ]; then echo "Error: DOCKER_PUSH_TARGET must be set"; exit 1; fi'
+      - docker buildx create --use
+      - docker buildx build --push 
+        --build-arg VERSION_FLAGS={{.VERSION_FLAGS}}ga Ωx
+        --build-arg CONTROLLER_APP={{.CONTROLLER_APP}}
+        --platform linux/amd64,linux/arm64 
+        --tag "{{.DOCKER_PUSH_TARGET}}/{{.CONTROLLER_DOCKER_IMAGE}}" .
+    status:
+      - "docker manifest inspect {{.LOCAL_REGISTRY_CONTROLLER_DOCKER_IMAGE}} > /dev/null"
```

Comment:
This `status` check doesn't seem to mesh with the commands being run. What is producing the `LOCAL_REGISTRY_CONTROLLER_DOCKER_IMAGE`? If nothing, this status check is wrong?

It should do a manifest inspect on an actual produced image maybe?

Reply:
or maybe we can delete this since we have `run: always`? Might need to read the taskfile docs on that one

Reply:
As I understand `go-task`, if you have `run: always` then it will always be run, regardless of the status checks. Either remove `run:` or `status:`, no need to have both.